classes/WP_CLI/ProgressBar.php

```diff
@@ -64,7 +84,7 @@ public function tick() {
 		$this->progress_bar->tick();
 		$this->total_ticks++;
 
-		do_action( 'action_scheduler/progress_tick', $this->total_ticks );
+		do_action_deprecated( 'action_scheduler/progress_tick', $this->total_ticks ); // phpcs:ignore WordPress.NamingConvetions.ValidHookName.UseUnderscores
```

Comment:
Is the switch to `do_action_deprecated` intentional and related to the larger [WP CLI-related PR](https://github.com/woocommerce/action-scheduler/pull/1071), can we/should we offer an explanation/supply a `$message` if so?

Reply:
I think I must've meant to add a hook that adheres to the PHPCS standards. Will add that now.